src/components/Tabs/index.js

```diff
@@ -23,6 +24,9 @@ export default class Tabs extends React.Component {
   handleClick(index, event) {
     event.preventDefault();
     this.setState({ selected: index });
+    if (typeof this.props.onChange === 'function') {
+      this.props.onChange(index);
```

Comment:
Should we send more than the index? Maybe returning the react component that is visible ?